anta/tests/interfaces.py

```diff
@@ -883,3 +883,134 @@ def test(self) -> None:
+            if not (interface_details := get_value(command.json_output, f"portChannels.{portchannel}.interfaces.{interface}")):
+                self.result.is_failure(f"Interface '{interface}' is not configured to be a member of LACP '{portchannel}'.")
+                continue
+
+            # Verify the interface is bundled in port channel.
+            actor_port_status = interface_details.get("actorPortStatus")
+            if actor_port_status != "bundled":
+                message = f"For Interface {interface}:\nExpected `bundled` as the `local port status`, but found `{actor_port_status}` instead.\n"
+                self.result.is_failure(message)
+                continue
+
+            # Collecting actor and partner port details
+            actor_port_details = interface_details.get("actorPortState", {})
+            partner_port_details = interface_details.get("partnerPortState", {})
+
+            # Collecting actual interface details
+            actual_interface_output = {
+                "actor_port_details": {
+                    "activity": actor_port_details.get("activity", "NotFound"),
+                    "aggregation": actor_port_details.get("aggregation", "NotFound"),
+                    "synchronization": actor_port_details.get("synchronization", "NotFound"),
+                    "collecting": actor_port_details.get("collecting", "NotFound"),
+                    "distributing": actor_port_details.get("distributing", "NotFound"),
+                    "timeout": actor_port_details.get("timeout", "NotFound"),
+                },
+                "partner_port_details": {
+                    "activity": partner_port_details.get("activity", "NotFound"),
+                    "aggregation": partner_port_details.get("aggregation", "NotFound"),
+                    "synchronization": partner_port_details.get("synchronization", "NotFound"),
+                    "collecting": partner_port_details.get("collecting", "NotFound"),
+                    "distributing": partner_port_details.get("distributing", "NotFound"),
+                    "timeout": partner_port_details.get("timeout", "NotFound"),
+                },
+            }
+
+            # Forming expected interface details
+            expected_interface_output = {
+                "actor_port_details": {
+                    "activity": True,
+                    "aggregation": True,
+                    "synchronization": True,
+                    "collecting": True,
+                    "distributing": True,
+                    "timeout": False,
+                },
+                "partner_port_details": {
+                    "activity": True,
+                    "aggregation": True,
+                    "synchronization": True,
+                    "collecting": True,
+                    "distributing": True,
+                    "timeout": False,
+                },
+            }
```

Comment:
Can we find a way to avoid repetition? For example, the dictionary of `actor_port_details` is the same as `partner_port_details`. We could use constants or helper functions, etc.

Reply:
Updated.